src/cmd/add.nim

```diff
@@ -81,11 +101,17 @@ proc paxAdd*(input: string, strategy: string): void =
   echoMod(cfMod, moreInfo = true)
   echo ""
 
-  returnIfNot promptYN("Are you sure you want to install this mod?", default = true)
+  returnIfNot promptYN("Are you sure you want to install this mod?",
+      default = true)
 
   echoInfo "Installing ", cfMod.name.cyanFg, ".."
-  manifest.installMod(cfMod.projectId, cfModFile.fileId)
+  let modToInstall = initManifestFile(cfMod.projectId, cfModFile.fileId,
+      cfMod.name, true, cfModFile.dependencies)
+  manifest.installMod(modToInstall)
+
+  if noDepends:
```

Comment:
Other way round, if `noDepends` is false you want to install dependencies, so it's `if not noDepends:`.